cmd/gapir/cc/main.cpp

```diff
@@ -57,13 +57,12 @@
 using namespace core;
 using namespace gapir;
 
-const size_t RAM_CACHE_SIZE = 256ul * 1024ul * 1024ul;
-
 namespace {
 
 std::shared_ptr<MemoryAllocator> createAllocator() {
+  size_t size =
```

Comment:
I would prefer:
```
#if defined(__x86_64) || defined(__aarch64__) which we use elsewhere.
size_t size = 16 * 1024ull * 1024ull;
#else
size_t size = 2 * 1024ull * 1024ull
#endif
```
`sizeof(void*) == 8` is non-standard (although I think is actually true for all of our compilers.

Reply:
I was thinking my sizeof(void*) < 8 would be more portable to future platforms. Don't mind making the change though!